packages/components/select/src/select.vue

```diff
@@ -35,81 +35,86 @@
+                  :size="collapseTagSize"
+                  :hit="item.hitState"
+                  :type="tagType"
+                  disable-transitions
+                  @close="deleteTag($event, item)"
                 >
-                  <template #default>
-                    <span :class="nsSelect.e('tags-text')"
-                      >+ {{ selected.length - 1 }}</span
-                    >
-                  </template>
-                  <template #content>
-                    <div :class="nsSelect.e('collapse-tags')">
-                      <div
-                        v-for="(item, idx) in selected.slice(1)"
-                        :key="idx"
-                        :class="nsSelect.e('collapse-tag')"
+                  <span :class="nsSelect.e('tags-text')" :style="tagTextStyle">
+                    {{ item.currentLabel }}
+                  </span>
+                </el-tag>
+                <el-tag
+                  v-if="selected.length > maxCollapseTags"
+                  :closable="false"
+                  :size="collapseTagSize"
+                  :type="tagType"
+                  disable-transitions
+                >
+                  <el-tooltip
+                    v-if="collapseTagsTooltip"
+                    :disabled="dropMenuVisible"
+                    :fallback-placements="['bottom', 'top', 'right', 'left']"
+                    :effect="effect"
+                    placement="bottom"
+                    :teleported="teleported"
+                  >
+                    <template #default>
+                      <span :class="nsSelect.e('tags-text')"
+                        >+ {{ selected.length - maxCollapseTags }}</span
                       >
-                        <el-tag
-                          :key="getValueKey(item)"
-                          class="in-tooltip"
-                          :closable="!selectDisabled && !item.isDisabled"
-                          :size="collapseTagSize"
-                          :hit="item.hitState"
-                          :type="tagType"
-                          disable-transitions
-                          :style="{ margin: '2px' }"
-                          @close="deleteTag($event, item)"
+                    </template>
+                    <template #content>
+                      <div :class="nsSelect.e('collapse-tags')">
+                        <div
+                          v-for="(item, idx) in selected.slice(maxCollapseTags)"
+                          :key="idx"
+                          :class="nsSelect.e('collapse-tag')"
                         >
-                          <span
-                            :class="nsSelect.e('tags-text')"
-                            :style="{
-                              maxWidth: inputWidth - 75 + 'px',
-                            }"
-                            >{{ item.currentLabel }}</span
+                          <el-tag
+                            :key="getValueKey(item)"
```

Comment:
**Suggestion:** I was thinking that we should move this `key` section to the `v-for` directive.

Reply:
Sure